app/src/androidTest/java/comcmput301f16t01/github/carrier/SearchingTests.java

```diff
@@ -72,61 +51,63 @@ private void chillabit( long time ) {
         }
     }
 
-    // TODO implement searchByLocation, this test will not pass
     public void testDriverSearchByLocation() {
```

Comment:
In comments could I have what you are testing, and if you test for any exceptional cases? 
Also maybe include reference to which use case this tests, if applicable.